docs/organizations/using-a-coding-standard.md

```diff
@@ -5,24 +5,27 @@
 
     For more information [read the release announcement](https://blog.codacy.com/organization-coding-standards/) or [watch a demo (3 min)](https://www.loom.com/share/19642d09662e40f2820bf2be6bdf3660) to learn how to create a coding standard for your organization.
 
-Create a coding standard on your organization to define and apply shared tool and code pattern configurations consistently across your repositories. You can also apply the coding standard to new repositories automatically by defining the coding standard as default.
+Create coding standards on your organization to define and apply shared tool and code pattern configurations consistently across your repositories. You can also apply a coding standard to new repositories automatically by defining it as default.
 
-The organization coding standard helps you ensure that Codacy analyzes multiple repositories with the same tool and code pattern settings. For example, you can use the organization coding standard to ensure that all your repositories and teams are following the same security rules or coding conventions.
+Coding standards help you ensure that Codacy analyzes multiple repositories with the same tool and code pattern configurations. For example, you can use a coding standard to ensure that a team or a group of repositories follow the same security rules or coding conventions.
 
-When you customize the tools or code patterns of a repository that follows the coding standard, Codacy warns you that the repository will stop following the coding standard and asks for your confirmation.
+When you customize the tools or code patterns of a repository that follows a coding standard, Codacy warns you that the repository will stop following the coding standard and asks for your confirmation.
 
```

Comment:
Maybe it's already understandable, but how about having something like this:

`When you customize the tools or code patterns, at the repository level, of a repository that`

Reply:
Do we need to link to the specific documentation?

Reply:
As a rule of thumb, we should use links frugally because each link is like a potential "fork" in the flow of the user reading the docs. The goal of this sentence is to explain that the repositories become "unliked" from the coding standard if there's a change to the tool and code pattern configurations of the repository. Adding a link to the other page here can actually detract from this goal.

Reply:
Seeing it this way it makes sense to keep the focus on the behaviour and not have a link.